### PR TITLE
Add a JJB template and a simple test script for use with CentOS CI.

### PR DESCRIPTION
**Pull request by bowlofeggs**
Signed-off-by: Randy Barlow <randy@electronsweatshop.com>

### CODE REVIEW

File: devel/ci/run_tests.sh
Comment (bstinsonmhk) [inline devel/ci/run_tests.sh]: You'll be getting a CentOS machine to start off with, so you'll want to use Yum

File: devel/ci/githubprb-project.yml
Comment (bstinsonmhk) [inline devel/ci/githubprb-project.yml]: Apologies for not calling this out too, you'll want to fill out this section. This should include you and perhaps some of the other bodhi maintainers.

You can also add an 'org-list' section, such that pull requests from anyone in the organization (fedora-infra for example) would automatically run tests on their PRs.

Anyone not in the admin-list (folks that can trigger tests, and whose PRs will be automatically tested) or the org-list (folks whose PRs will be automatically tested), will be held until an admin asks for a test in the github comments.

File: devel/ci/run_tests.sh
Comment (bowlofeggs) [reply devel/ci/run_tests.sh]: Ah yes, muscle memory - thanks for noting!

**Comment by bowlofeggs** [comment]
@bstinsonmhk I believe I have addressed your comments now, thanks!

**Comment by bstinsonmhk** [comment]
:+1: once this is merged, i'll do the service job

**Review by jeremycline — APPROVED** [review]
I know basically nothing about JJB, but since it's CI stuff the proof is in the pudding so I say go for it :shipit:

**Comment by bowlofeggs** [comment]
@bstinsonmhk This is now merged - can you set up the service job for me? Thanks so much, this is very exciting!

**Comment by bowlofeggs** [comment]
I filed https://bugs.centos.org/view.php?id=13406 to request a service job for running the tests on CentOS's infrastructure.

**Comment by bstinsonmhk** [comment]
All set: https://ci.centos.org/job/bodhi-bodhi/

That should start triggering on new PRs shortly.

**Comment by bowlofeggs** [comment]
It looks like I need to give @centos-ci write access to the repo, according to:

https://ci.centos.org/job/bodhi-bodhi/lastBuild/console

**Comment by bowlofeggs** [comment]
@bstinsonmhk I invited @centos-ci to have write access to this repository as per that error message, but I imagine someone has to accept the write access for that account?

**Pull request merged**
